ssl_league_msgs/msg/game_controller/game_events/TooManyRobots.msg

```diff
@@ -0,0 +1,4 @@
+ssl_league_msgs/Team by_team
+int32[<=1] num_robots_allowed
+int32[<=1] num_robots_on_field
+geometry_msgs/Point32[<=1] ball_location
```

Comment:
What does the `[<=1]` mean in this context? Is it an array of length 1 or fewer similar to `optional` in proto?

Reply:
Exactly. It's an admittedly pretty kludgy way to imitate optional fields.

Reply:
Makes sense, I'm down for that